data/members/adelinocosta.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Everyone has their own file? Seems a bit excessive imo. But alright.

Reply:
This way you can add/remove members easily, for example the two translations members from ILHC.

Reply:
I guess it's mostly a matter of personal opinion. But your approach might indeed be best in this case.
Cool 😄